tests/utils/tiny_model_summary.json

```diff
@@ -1718,7 +1718,7 @@
         "model_classes": [
             "EfficientNetForImageClassification"
         ],
-        "sha": "6ed195ee636d2c0b885139da8c7b45d57ebaeee0"
+        "sha": "993d088cf937b8a90b61f68677cd8f261321c745"
```

Comment:
Corresponds to [this commit](https://huggingface.co/hf-internal-testing/tiny-random-EfficientNetForImageClassification/commit/993d088cf937b8a90b61f68677cd8f261321c745).

This is the processor config that results from the corrected workflow.